www/src/pages/blog/astro-021-preview.astro

```diff
@@ -0,0 +1,119 @@
+---
+import { Markdown, Code } from 'astro/components';
+import BaseHead from '../../components/BaseHead.astro';
+import BlogHeader from '../../components/BlogHeader.astro';
+import BlogPost from '../../components/BlogPost.astro';
+import GoogleAnalytics from '../../components/GoogleAnalytics.astro';
+
+let title = 'Astro 0.21 Preview: Vite + WASM = ⚡️';
+let description = `Get a sneak preview of what's next for Astro, including our new Vite build engine and WASM-powered Go compiler.`;
+let publishDate = 'September 23, 2021';
+let permalink = 'https://astro.build/blog/astro-021-preview';
+let lang = 'en';
+---
+
+<html lang={ lang ?? 'en' }>
+  <head>
+    <BaseHead title={title} description={description} canonicalURL={permalink} image="https://astro.build/assets/blog/demo-day-2021-09/social.png" />
+    <link rel="stylesheet" href={Astro.resolve('../../scss/blog.css')} />
+    <style>
+      .chat-messages {
+
+      }
+    </style>
+  </head>
+
+  <body>
+    <BlogHeader />
+    <BlogPost title={title} author="fred" heroImage="/assets/blog/demo-day-2021-09/hero.png" publishDate={publishDate}>
+      <Markdown>
+        Astro v0.21.0 will be our biggest release yet. At a high-level, it includes:
+
+        - [A new build engine, powered by Vite](#hello-vite)
+        - [A new WASM compiler, written in Go](#hello-wasm)
+        - [Brand new features, like Components-in-Markdown](#components-in-markdown)
+        - [A new system for HTML live-updating via HMR](#hmr-meet-html)
+
+        You can try out this release today on StackBlitz, or by running `npm init astro --next` in a new directory on your machine.
```

Comment:
TODO: create a `@next` demo on Stackblitz

Reply:
I would argue for codesandbox or something that is more accessible since stackblitz's node container is only supported on Chrome and Edge, not Safari or Firefox